easybuild/tools/job/gc3pie.py

```diff
@@ -133,7 +134,7 @@ def init(self):
         if cfgfile:
             self.config_files.append(cfgfile)
 
-        # additional subdirectory, since GC3Pie cleans up the output dir?!
+        # additional subdirectory, since GC3Pie cleans up the output dir!
```

Comment:
I know this is just a comment, but GC3Pie does not *clean* the output dir: if it exists, it moves the exiting one to a backup name and creates a new output directory.